model-optimizer/requirements.txt

```diff
@@ -8,3 +8,4 @@ onnx>=1.8.1
 defusedxml>=0.7.1
 urllib3>=1.26.4
 requests>=2.25.1
+fastjsonschema~=2.15.1
```

Comment:
did you check if this package exists on all OSes?

Reply:
According to package info It is OS Independent.